README.md

```diff
@@ -39,77 +102,8 @@ under `$GOPATH/github.com/openshift/origin-aggregated-logging` you can use its `
 to build the logging, elasticsearch-operator, and cluster-logging-operator images; deploy a 4.x cluster;
 push the images to the cluster; deploy logging; and launch the logging CI tests.
 
+## Updating Dependencies
 
-### Make targets
-
-Make targets use commands such as `oc`, `imagebuilder` and `operator-sdk`.  If a
-suitable command is not found in `./bin:$PATH`, `make` will automatically build
-or download the command in `./bin`.
-
-See also: [sdk_setup.sh](https://raw.githubusercontent.com/openshift/origin-aggregated-logging/master/hack/sdk_setup.sh) to set up your environment.
-
-#### Targets that do not require a cluster
-
-|Target|Description|
-|-----|------------------------------------------|
-|`all`|Run all local tests, lint and format code.|
-|`lint`|Lint and format code.|
-|`test-unit`|Run unit tests.|
-|`image`|Build the image but do not deploy it.|
-
-#### Targets that require a cluster
-
-These targets assume:
-* Logged into an OKD cluster with cluster role `cluster-admin`
-* Clone of `openshift/elasticsearch-operator` repository in `$GOPATH`
-
-The deployment can be optionally modified using any of the following:
-
-| Env Var | Default | Description|
-|---------|---------|------------|
-|`EXCLUSIONS`|none|The list of manifest files that should will be ignored|
-|`PUSH_USER`|none|The name of the user e.g. `kubeadmin` used to push images to the remote registry|
-|`PUSH_PASSWORD`|none|The password for `PUSH_USER`|
```

Comment:
These variables will be essential after merging #511. We need a section for them.

Reply:
Can you LGTM this and add the section for the PUSH variables in a later PR? I don't want to mis-state the story.

Reply:
i closed out #511 -- i think we should be using the `oc login -u kubeadmin -p <password>` approach